doc/tutorials/vuart_configuration.rst

```diff
@@ -12,7 +12,7 @@ only works on a single function.
 
 Only two vUART configurations are added to the predefined scenarios,
 but you can customize the scenarios to enable more using the :ref:`ACRN
-configuration toolset <acrn_config_workflow>`.
+configurator tool <acrn_configurator_tool>`.
 
```

Comment:
Starting at line 241, this file also refers to "configuration tools" (example: "Setup ACRN vUART Using Configuration Tools"). Should I change these to "ACRN configurator tool"?

Reply:
If we're making this name change, we should do it everywhere including titles of documents if the reference is to the configurator tool itself (and not a generic mention of configuration tools).

Reply:
@gvancuts @NanlinXie In this file, does "configuration tools" refer to only the configurator tool or other tools? 
@dbkinder Regarding your comment, I did a search for "configuration tool" and "configuration editor" which led me to edit these files. Can you provide an example of other places that should be changed?

Reply:
I think you've got them covered now.